### PR TITLE
moving API doc from the README to the source code

### PR DESCRIPTION
**Pull request by TornaxO7**

### CODE REVIEW

**Comment by TornaxO7** [comment]
this kind of doc should be in the code and not in the README in my opinion.

File: src/lib.rs [hunk 1]
```diff
@@ -1,6 +1,6 @@
 mod image_utils;
 
-use anyhow::{anyhow, Result};
+use anyhow::{anyhsow, Result};
```
Comment (nashaofu) [inline src/lib.rs:3]: Is there a mistake with this？

Comment (TornaxO7) [reply src/lib.rs:3]: oh eh, yes

Comment (TornaxO7) [reply src/lib.rs]: fixed! :)

**Review by nashaofu — APPROVED** [review]

**Pull request merged**